### ISSUE TITLE
Missing/Needed Art

### ISSUE THREAD

**Issue by BraedonWooding** [OrderOfThePorcupine/ProjectPorcupine#101]
Just a list of needed/missing art, if there are any extras post them as comments below and i'll add them to this list;
- Better doors
- Berth
- More variations in walls
- Sprites for things like currencies, and resources?

**Comment by koosemose** [comment]
Doors are just fine, the only thing that might be useful is having minor variation on the airlock door, having a red/green indicator light (to be used by the door to indicate pressure equalization). 

Resources honestly could use better sprites, the "Everything sitting on trays" is bad imo.

I personally  don't think traingular surfaces are really needed, we are a tile based game and they would serve no purpose other than purely aesthetic. 

Most of our machine sprites need smaller versions they are all unreasonable large, the whole "conveyor belt" input thing needs to go, and will, we just need to get internal storage on the machines (and appropriate UI for showing it, but we should get appropriate sprite art for it.

A Build Icon for the Airlock RoomBehavior is needed, it's still using my bad programmer art, I'm not sure what a good icon would be to indicate this, so it would take some creativity on a spriter's part.

We need some sprites for the minable resource that can be properly integrated into the wall graphics, the current "Crystal growing on top" design, while the art is pretty, looks ridiculous.

And we need an unsuited variation of the character sprite (my limited spriting ability has only got me as far as a front view), I can provide the source files for the suited sprite and what I managed to get for unsuited (source is in aseprite file format), for a base for a spriter.

**Comment by BraedonWooding** [comment]
I only added doors since in the door.png file it said that they needed to be replaced, will specify .  

Yep the triangular were mainly just for aesthetic which I think is still quite important since you are building a space station and having the ability to make it look a little unique for you is important imo.

I agree with the rest, though I honestly quite like the mineable resource graphic (mainly just because it's pretty) so I would put that lower on a 'priority list' since replacing coding art first is key

**Comment by koosemose** [comment]
door.png *has* been replaced and isn't used anywhere, it just has managed to escape notice as still existing until now.

And I agree the minable resource graphic is pretty, but it's also horribly wrong for what it is. We could keep the graphic around for reuse on something more appropriate, something that would actually feasibly have a crystal like structure, and grow up from the floor, rather than how it's currently used, growing from the top of a wall.

We only actually have 2 pieces of programmer art atm, that being the Airlock RoomBehavior and the Berth.

**Comment by BraedonWooding** [comment]
Ahhh yep good that you got rid of it :D.  Yeah I actually agree with the thing about the resource graphic (and you were talking about a rare resource this could be a good stand in for that), yeah we only have two pieces of programmer art (thought we had the water stuff but then remembered that was replaced 'ages' ago) but we also have a bunch of things with wrong angles and so on that also need a little touchup :D.

**Comment by koosemose** [comment]
Yeah, lots of stuff with "wrong" graphics in some way or another. Either wrong angle, too big, or consisting of elements we don't want in the end.
